### PR TITLE
Pods 4885

### PR DESCRIPTION
**Pull request by vinnicombe**

### CODE REVIEW

File: app/repositories/AdminDataRepository.scala [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+class AdminDataRepository @Inject()(
+                                     mongoComponent: ReactiveMongoComponent,
+                                     configuration: Configuration
+                                   )(implicit val ec: ExecutionContext) extends ReactiveRepository[JsValue, BSONObjectID](
+  configuration.get[String](path = "mongodb.pension-administrator-cache.admin-data.name"),
+  mongoComponent.mongoConnector.db,
+  implicitly
+) {
+
+  val featureToggleDocumentId = "toggles"
```
Comment (grantrobson) [inline app/repositories/AdminDataRepository.scala:41]: Make these private

File: app/service/FeatureToggleService.scala [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+import scala.concurrent.duration.{Duration, FiniteDuration, SECONDS => Seconds}
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class FeatureToggleService @Inject()(
+                                      adminDataRepository: AdminDataRepository,
+                                      cacheApi: AsyncCacheApi
+                                    )(implicit ec: ExecutionContext) {
+  val cacheValidFor: FiniteDuration = Duration(2, Seconds)
```
Comment (grantrobson) [inline app/service/FeatureToggleService.scala:34]: Make private

**Review by grantrobson — APPROVED** [review]

File: app/repositories/AdminDataRepository.scala (hunk 1 above)
Comment (vinnicombe) [reply app/repositories/AdminDataRepository.scala]: Done

File: app/service/FeatureToggleService.scala (hunk 2 above)
Comment (vinnicombe) [reply app/service/FeatureToggleService.scala]: Done

**Pull request merged**